run-tests.sh

```diff
@@ -83,53 +81,77 @@ check_commitlint () {
     fi
 }
 
-check_shellcheck () {
+check_shellcheck() {
     find . -name "*.sh" -exec shellcheck {} \+
 }
 
-check_pydocstyle () {
+check_pydocstyle() {
     pydocstyle reana_workflow_controller
 }
 
-check_black () {
+check_black() {
     black --check .
 }
 
-check_flake8 () {
+check_flake8() {
     flake8 .
 }
 
-check_openapi_spec () {
+check_openapi_spec() {
     FLASK_APP=reana_workflow_controller/app.py python ./scripts/generate_openapi_spec.py
     diff -q -w temp_openapi.json docs/openapi.json
     rm temp_openapi.json
 }
 
-check_manifest () {
+check_manifest() {
     check-manifest
 }
 
-check_sphinx () {
+check_sphinx() {
     sphinx-build -qnNW docs docs/_build/html
 }
 
-check_pytest () {
+check_helm() {
+    helm lint helm/reana
+}
+
+check_yamllint() {
+    yamllint .
+}
+
+check_markdownlint() {
```

Comment:
The `markdownlint` and friends should come in a separate commit with their own dot files... Please remove them from here for now, keeping the change only about `shfmt` matters.